scripts/update-scripts/update-scripts.js

```diff
@@ -0,0 +1,79 @@
+const fetch = require('node-fetch')
+const fs = require('fs')
+const path = require('path')
+const getSessionToken = require('../../helpers/get-session-token')
+//const updateNode = require('./update-node')
+
+const updateScripts = async (argv) => {
+  const { realm } = argv
+
+  // Check environment variables
+  const { FIDC_URL, PHASE = '0' } = process.env
+
+  if (!FIDC_URL) {
+    console.error('Missing FIDC_URL environment variable')
+    return process.exit(1)
+  }
+
+  try {
+    const sessionToken = await getSessionToken(argv)
+    console.log(`Using phase ${PHASE} config`)
+
+    // Read auth tree JSON files
+    const dir = path.resolve(
+      __dirname,
+      `../../config/phase-${PHASE}/am-scripts`
+    )
+
+    const scriptFileContent = fs
+      .readdirSync(dir)
+      .filter((name) => path.extname(name) === '.json') // Filter out any non JSON files
+      .map((filename) => require(path.join(dir, filename))) // Map JSON file content to an array
+
+    // Update each script
+    await Promise.all(
+        scriptFileContent.map(async (scriptFile) => {
+            const baseUrl = `${FIDC_URL}/am/json${realm}`
+            await Promise.all(
+                scriptFile.scripts.map(async (script) => {
+                    if (!script.payload._id) {
+                        return Promise.reject(new Error('Missing _id in script config'))
+                    }
+                    //updates the script content with encoded file
+                    script.payload.script = fs.readFileSync(`./config/phase-${PHASE}/am-scripts/scripts-content/${script.filename}`, {encoding: 'base64'});
```

Comment:
I would use the `dir` variable here as the beginning of the path. That way the path will resolve if the script is called from a different directory.